app/javascript/components/SearchQueryDisplay.js

```diff
@@ -68,7 +91,7 @@ export default function SearchQueryDisplay({terms, facets}) {
   }
   return (
     <div className="search-query">
-      <FontAwesomeIcon icon={faSearch} />: {termsDisplay}{facetsDisplay}
+      <FontAwesomeIcon icon={faSearch} />: {termsDisplay}{facetsDisplay} <ClearAllButton/>
```

Comment:
```suggestion
      <FontAwesomeIcon icon={faSearch} />: <span className="query-text">{termsDisplay}{facetsDisplay}</span> <ClearAllButton/>
```
Then change the selectors in the tests so that you're only reading the .text() from the query-text span